lib/test-endpoint.sh

```diff
@@ -2,13 +2,26 @@
 
 unset DD_LOGS_VALID_ENDPOINT
 DATADOG_DIR="${DATADOG_DIR:-/home/vcap/app/datadog}"
+DD_US_API_SITE="https://api.datadoghq.eu/api/"
+DD_EU_API_SITE="https://api.datadoghq.us/api/"
+DD_API_SITE=${DD_US_API_SITE}
+DD_USE_EU=false
+
+if [[ $DD_SITE == *".eu"* ]]; then
+  DD_USE_EU=true
+  DD_API_SITE=${DD_EU_API_SITE}
+fi
 
 if [ -z $DD_LOGS_CONFIG_LOGS_DD_URL ]; then
   # Initialize to default value
+  # If DD_SITE contains ".eu" use the EU endpoint, otherwise default to US
   if [ -n $DD_LOGS_CONFIG_DD_PORT -a -n $DD_LOGS_CONFIG_DD_URL ]; then
     DD_LOGS_CONFIG_LOGS_DD_URL="$DD_LOGS_CONFIG_DD_URL:$DD_LOGS_CONFIG_DD_PORT"
   else
-    DD_LOGS_CONFIG_LOGS_DD_URL="agent-intake.logs.datadoghq.com:10516"
+    if [[ $DD_USE_EU ]]; then
+      DD_LOGS_CONFIG_LOGS_DD_URL="agent-intake.logs.datadoghq.eu:443"
```

Comment:
Is it 443 in EU ? Seems to be the http endpoint no ?

Reply:
It's actually `443` in EU indeed: https://docs.datadoghq.com/logs/log_collection/?tab=eusite#datadog-logs-endpoints and `10516` in US: https://docs.datadoghq.com/logs/log_collection/?tab=ussite#datadog-logs-endpoints